snabbdom.js

```diff
@@ -8,8 +8,9 @@ var is = require('./is');
 function isUndef(s) { return s === undefined; }
 function isDef(s) { return s !== undefined; }
 
+// deal with case sensivity better than that
```

Comment:
I left this comment because people can also use uppercase tags in their views. I guess we should lowercase the tag after removing classes and ids.